vowpal_wabbit/vw-alpine.Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM vowpalwabbit/ubuntu1604-build AS build
-ARG branch_or_tag
+ARG branch_or_tag=8.8.0
```

Comment:
Is this how you do default values?

Reply:
yea, not a fan of that being the default value but thought it was better than master

Reply:
Makes sense